armory/datasets/adversarial/carla_obj_det_dev/carla_obj_det_dev.py

```diff
@@ -0,0 +1,225 @@
+"""carla_obj_det_dev dataset."""
```

Comment:
I built from lucas's fork to get the scenario_config changes, but did not realize these changes weren't merged as well, should I rebase ontop of tfdsv4 instead of from [lc/tfdsv4-fix-configs](https://github.com/lcadalzo/armory/tree/tfdsv4-fix-carla-configs)?

Reply:
Addressed with #1836